### ISSUE TITLE
Integration: Elementor

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#582]
## Describe the solution you'd like

There should be a way to view Accessibility Checker reports in the Elementor editor, similar to how Yoast can be viewed in Elementor. [See Basecamp convo here.](https://3.basecamp.com/3579237/buckets/29333825/chats/5333625436@7297229143)

## Why do you think this feature is something we should consider for this plugin?
Elementor is used on many websites. 😀

**Comment by SteveJonesDev** [comment]
Move to Basecamp
